### PR TITLE
feat(unlock-app) - add email from aidrop page

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
This PR let the lock manager save an email address before airdropping the key so that the recipient receives an email once the key has been minted.

# Issues

Fixes #
Refs #

- [x]  adds email in from 
Screenshot 2022-07-22 at 15 22 19

- [x] email visible as metadata after minting 
Screenshot 2022-07-22 at 15 25 35

- [x] confirmation email after mining 
Screenshot 2022-07-22 at 15 26 38

- [x] new UI airdrop 
Screenshot 2022-07-22 at 19 38 20

- [x] new UI airdrop with errors 
Screenshot 2022-07-22 at 19 39 11

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/creator/members/GrantKeysDrawer.tsx [hunk 1]
```diff
@@ -212,6 +239,26 @@ const GrantKeyForm = ({ onGranted, lock }: GrantKeyFormProps) => {
         </div>
       </div>
 
+      <div className="flex flex-wrap mb-6 -mx-3">
+        <div className="w-full px-3">
+          <Label htmlFor="grid-recipient">Email</Label>
+          <Input
```
Comment (searchableguy) [inline unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:245]: Wanna use our UI inputs here instead for consistency?

Comment (kalidiagne) [reply unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:245]: yes! i am planning to do it, but in a different PR

Comment (julien51) [reply unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:245]: Any reason to _not_ include it here?

Comment (kalidiagne) [reply unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:245]: oh, not at all, just to keep things separate :)

Comment (kalidiagne) [reply unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:238]: just updated! with new UI

File: unlock-app/src/components/creator/members/GrantKeysDrawer.tsx [hunk 2]
```diff
@@ -252,23 +249,19 @@ const GrantKeyForm = ({ onGranted, lock }: GrantKeyFormProps) => {
             })}
           />
           {errors.recipient && (
-            <p className="text-xs -mt-4 text-[#f24c15]">
+            <p className="text-xs text-[#f24c15]">
```
Comment (julien51) [inline unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:252]: This error message is not correct. Also are you validating email addresses?

File: unlock-app/src/components/creator/members/GrantKeysDrawer.tsx [hunk 3]
```diff
@@ -252,23 +249,19 @@ const GrantKeyForm = ({ onGranted, lock }: GrantKeyFormProps) => {
...
               Please make sure you enter a valid Ethereum address
             </p>
           )}
         </div>
```
Comment (julien51) [inline unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:256]: Let's add a little explanation, similar to the one we have for the key manager:
"If you enter the recipient's email address, they will receive an email to confirm the airdrop, as well as a link to view their NFT. The email will also include a QR code of the signed NFT that they can use to prove that they own it in when attending in person events."

File: unlock-app/src/components/interface/Drawer.tsx [hunk 4]
```diff
@@ -51,10 +51,10 @@ export const Drawer = ({
               <div className="relative w-screen max-w-md">
                 <Transition.Child {...easeOutTransaction}>
                   <div className="absolute top-0 right-0 h-12 w-12 p-4">
-                    <Close size="16px" onClick={() => setIsOpen(false)} />
+                    <CloseIcon size="25px" onClick={() => setIsOpen(false)} />
                   </div>
                 </Transition.Child>
-                <div className="h-full flex flex-col py-6 bg-white shadow-xl overflow-y-scroll">
+                <div className="h-full flex flex-col py-6 bg-[#FFFDF8] shadow-xl overflow-y-scroll">
```
Comment (julien51) [inline unlock-app/src/components/interface/Drawer.tsx:57]: Do we want to use a variable for that?

**Review by julien51 — COMMENTED** [review]
This is looking very promising.

File: unlock-app/src/components/creator/members/GrantKeysDrawer.tsx (hunk 2 above)
Comment (kalidiagne) [reply unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:252]: oh! thk! 
actually, the input is 'email type', it take basic form validation, also changed with the correct error!

File: unlock-app/src/components/interface/Drawer.tsx (hunk 4 above)
Comment (kalidiagne) [reply unlock-app/src/components/interface/Drawer.tsx]: added as brand color in tailwind config, for dashboard refactoring, we need it a lot

File: unlock-app/src/components/creator/members/GrantKeysDrawer.tsx (hunk 3 above)
Comment (kalidiagne) [reply unlock-app/src/components/creator/members/GrantKeysDrawer.tsx:256]: for email a

> This is looking very promising.

Updated with small changes in comments!

**Review by julien51 — APPROVED** [review]

**Pull request merged**